Tools/customize.h

```diff
@@ -108,22 +108,22 @@ namespace AnaConsts{
    const float defaultHTcut = 300;
 
 //                                    minAbsEta, maxAbsEta, minPt, maxPt,   maxIso,  maxMtw
-   const IsoAccRec     muonsArr =    {   -1,       2.4,      10,     -1,       0.2,     -1  };
-   const IsoAccRec muonsMiniIsoArr = {   -1,       2.4,      5,     -1,       0.2,     -1  };
-   const IsoAccRec muonsTrigArr =    {   -1,       2.4,       5,     -1,       0.4,     -1  };
-//   const IsoAccRec   isoTrksArr =    {   -1,        -1,      10,     -1,       0.1,    100  };
-   const IsoAccRec   isoLepTrksArr = {   -1,       2.5,       5,     -1,       0.2,    100  };
-   const IsoAccRec   isoHadTrksArr = {   -1,       2.5,      10,     -1,       0.1,    100  };
-//   const IsoAccRec   isoTrksArr =    {   -1,        -1,     10,     -1,       0.1,    120  };
+   const IsoAccRec muonsArr        = {   -1,       2.4,      10,     -1,       0.2,     -1  };
+   const IsoAccRec muonsMiniIsoArr = {   -1,       2.4,       5,     -1,       0.2,     -1  };
+   const IsoAccRec muonsTrigArr    = {   -1,       2.4,       5,     -1,       0.4,     -1  };
+// const IsoAccRec isoTrksArr      = {   -1,        -1,      10,     -1,       0.1,    100  };
```

Comment:
this may be lined up in comment, but it will not be lined up if it were uncommented ... just leave alignment like this alone